packages/concerto-cto/lib/printer.js

```diff
@@ -104,6 +104,9 @@ function typeFromMetaModel(mm){
     case `${MetaModelNamespace}.ObjectProperty`:
         result += ` ${mm.type.name}`;
         break;
+    case `${MetaModelNamespace}.MapProperty`:
```

Comment:
These are not properties, in the sense we use it for the properties of a class (a name and a type, and meta-properties). These are the definitions of the type for the key and the type for the value.